Tropos/Sources/Categories/UIApplication+TRReactiveBackgroundTask.h

```diff
@@ -0,0 +1,8 @@
+#import <UIKit/UIKit.h>
+#import <ReactiveCocoa/ReactiveCocoa.h>
+
+@interface UIApplication (TRReactiveBackgroundTask)
+
+- (RACSignal *)tr_backgroundTaskWithSignal:(RACSignal *(^)(void))signalBlock;
```

Comment:
Would you mind an attribution link to http://sharplet.me/post/123363708110/wrapping-uikit-background-tasks-with?